reducers/index.js

```diff
@@ -28,17 +28,16 @@ function flashcards(state = {}, action) {
       };
     case DEL_ALL:
       return initState;
-
     case DEL_DECK:
-      const newState = Object.keys(state).reduce((newSt, key) => {
+      return Object.keys(state).reduce((newSt, key) => {
         if (key !== action.deckName) {
-          newSt[key] = state[key];
+          return {
+            ...newSt,
+            [key]: state[key]
+          }
```

Comment:
Here it's just a cleaner way of returning the new state, without modifying directly the object `newSt`. Instead, we are returning a new copy on every iteration of the `reduce` method.